users/api/endpoint.go

```diff
@@ -93,6 +93,20 @@ func viewUserEndpoint(svc users.Service) endpoint.Endpoint {
 	}
 }
 
+func listUsersEndpoint(svc users.Service) endpoint.Endpoint {
```

Comment:
I am not quite sure, but I am leaning more towards `getUsers()`, because we have `update` below. Again - not quite sure. @dusanb94 @nmarcetic opinions?

Reply:
Looks actually that this terminology is used in other services, so for consistency it makes sense.

Reply:
This naming is used for any endpoint that returns a list. For example `listThingsEndpoint`

Reply:
@drasko Makes sense, but we use `list` in the other services. I think that the only reason is that `viewEntity` and `viewEntitites` are too much alike, so it's a bit more readable with `list`. Other than that, I don't see any reasons for using `list` instead of `get`.